pkg/hook/accessrequesthook/accessrequesthook.go

```diff
@@ -154,6 +150,24 @@ func (h Hook) NoAccess(ctx context.Context, profile *cfaws.Profile) (retry bool,
 	si.Writer = os.Stderr
 	si.Start()
 
+	if validation != nil && validation.HasReason {
+		var customReason string
+		msg := "Reason for access (Required)"
+		reasonPrompt := &survey.Input{
+			Message: msg,
+			Help:    "Will be stored in audit trails and associated with you",
```

Comment:
Was the wording intentional? Sounded a little funny to me
```suggestion
			Help:    "Will be stored in audit trails and associated with your request",
```